cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlCacheConfiguration.kt

```diff
@@ -161,6 +163,17 @@ class SqlCacheConfiguration {
   ): SqlCleanupStaleOnDemandCachesAgent =
     SqlCleanupStaleOnDemandCachesAgent(applicationContext, registry, clock)
 
+  @Bean
+  @ConditionalOnProperty(value = ["sql.cache-cleanup-agent.enabled"], matchIfMissing = true)
```

Comment:
Not sure if this should be on by default, at least initially. Could be flipped to on by default once demonstrated to be non-disruptive to installs that don't need it, or in the second OSS release that contains the cleanup agent, with enough opportunity to advise a config change.

Reply:
Good point - I like this rollout strategy. Thanks!